test-resources-plugin/src/main/java/io/micronaut/gradle/testresources/MicronautTestResourcesConsumerPlugin.java

```diff
@@ -39,23 +37,36 @@ public class MicronautTestResourcesConsumerPlugin implements Plugin<Project> {
     public void apply(Project project) {
         PluginManager pluginManager = project.getPluginManager();
         pluginManager.apply(MicronautBasePlugin.class);
-        TestResourcesConsumerConfiguration testResourcesExtension = createTestResourcesExtension(project);
-        pluginManager.withPlugin("io.micronaut.component", unused -> {
-            project.afterEvaluate(p -> p.getConfigurations().all(cnf -> configureDependencies(p, testResourcesExtension, project.getDependencies(), cnf)));
+        Configuration testResourcesConfiguration = createTestResourcesExtension(project);
+        pluginManager.withPlugin("io.micronaut.component", unused ->
+                project.afterEvaluate(p -> p.getConfigurations().all(cnf -> configureDependencies(testResourcesConfiguration, cnf)))
+        );
+    }
+
+    private Configuration createTestResourcesExtension(Project project) {
+        return project.getConfigurations().create(MicronautTestResourcesPlugin.TESTRESOURCES_CONFIGURATION, conf -> {
+            conf.setCanBeConsumed(false);
+            conf.setCanBeResolved(false);
+            conf.setDescription("Used to declare projects which provide test resources");
```

Comment:
"test resources" -> "micronaut test resources"?

Reply:
Technically speaking it's not specific to Micronaut: the same server could be used in a Spring app, if they implemented a resolution mechanism.